tools/pd-ctl/pdctl/command/global.go

```diff
@@ -55,23 +55,20 @@ var PDCli pd.Client
 
 func requirePDClient(cmd *cobra.Command, _ []string) error {
 	var (
-		caPath string
-		err    error
+		tlsConfig *tls.Config
+		err       error
 	)
-	caPath, err = cmd.Flags().GetString("cacert")
-	if err == nil && len(caPath) != 0 {
-		var certPath, keyPath string
-		certPath, err = cmd.Flags().GetString("cert")
-		if err != nil {
-			return err
-		}
-		keyPath, err = cmd.Flags().GetString("key")
-		if err != nil {
+	tlsConfig, err = parseTLSConfig(cmd)
+	if err != nil {
+		return err
+	}
+	if tlsConfig != nil {
+		if err = initHTTPSClient(tlsConfig); err != nil {
 			return err
 		}
```

Comment:
```suggestion
		return initHTTPSClient(tlsConfig)
```

Reply:
it need to execute `initNewPDClient` yet